tests/conftest.py

```diff
@@ -201,6 +202,7 @@ def pipx_temp_env(tmp_path, monkeypatch, pipx_session_shared_dir, request, utils
     seamless.
     """
     pipx_temp_env_helper(pipx_session_shared_dir, tmp_path, monkeypatch, request, utils_temp_dir, pipx_local_pypiserver)
+    paths.ctx.make_local()
```

Comment:
Why did you add this?

Reply:
The `OVERRIDE_PIPX_*` variables are only read and used during `make_local()` and `make_global()` calls. Simply monkeypatching these variables above does in itself not change anything on the `paths.ctx` object. Instead these changes are only "committed" during this `make_local()` call here.

Similarly to the [comment on `shared_libs`](https://github.com/pypa/pipx/pull/1476/files#r1694114472) this in mainly caused by the fact that `paths` uses a mutable global object that is initialized during module load (before any testing code can interfere) and then modified afterwards through `make_local()` and `make_global()` calls.

Reply:
That's true, and regarding my above comment, this invocation should even be _inside_ `pipx_temp_env_helper`.

Reply:
Oh yeah! Fixed in 4609f25.